### PR TITLE
Update LanguageModel notebook with Epochs

### PR DESCRIPTION
**Pull request by xihui-wu**
As part of https://github.com/tensorflow/swift-models/issues/592

### CODE REVIEW

File: Examples/Notebooks/LanguageModel.ipynb [hunk 1]
```diff
@@ -1,799 +1,1107 @@
...
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "let inBatches = dataset.inBatches(of: 4).lazy.map {\n",
+        "      (\n",
+        "        data: Tensor<Int32>($0.map(\\.first)),\n",
+        "        label: Tensor<Int32>($0.map(\\.second))\n",
+        "      )\n",
+        "    }"
```
Comment (BradLarson) [inline Examples/Notebooks/LanguageModel.ipynb:604]: The indentation here is a little weird. The last four lines could be moved a bit to the left.

File: Examples/Notebooks/LanguageModel.ipynb [hunk 2]
```diff
@@ -1,799 +1,1107 @@
...
+      },
+      "source": [
+        "dataset.shuffle()\n",
+        "\n",
+        "let inBatches = dataset.inBatches(of: 4).lazy.map {\n",
+        "      (\n",
+        "        data: Tensor<Int32>($0.map(\\.first)),\n",
+        "        label: Tensor<Int32>($0.map(\\.second))\n",
+        "      )\n",
+        "    }"
```
Comment (BradLarson) [inline Examples/Notebooks/LanguageModel.ipynb:684]: Same deal as above, looks like the indentation didn't line up right with the text box.

File: Examples/Notebooks/LanguageModel.ipynb [hunk 3]
```diff
@@ -1,799 +1,1107 @@
...
+        "        batchSize: 64, \n",
+        "        entropy: SystemRandomNumberGenerator()\n",
+        "      ).lazy.map { (batches: Batches) -> LazyMapSequence<Batches, LabeledTextBatch> in\n",
+        "        batches.lazy.map {\n",
+        "          (\n",
+        "            data: Tensor<Int32>($0.map(\\.first)),\n",
+        "            label: Tensor<Int32>($0.map(\\.second))\n",
+        "          )\n",
+        "        }\n",
+        "      }"
```
Comment (BradLarson) [inline Examples/Notebooks/LanguageModel.ipynb:834]: I think the indentation of this last part could be shifted to the left. I think the final curly bracket should align with the `l` on `let`.

Comment (xihui-wu) [reply Examples/Notebooks/LanguageModel.ipynb]: Got it

**Pull request merged**